app/views/admin/complete_persons.html.erb

```diff
@@ -1,18 +1,18 @@
 <% provide(:title, 'Finish unfinished persons') %>
 
 <div class="container">
-  <% competition_ids = @finish_persons.competition_ids %>
+  <% competition_ids = @competition_ids %>
```

Comment:
This line is redundant now, if you already have a `@competition_ids` variable then use it everywhere.

Reply:
No need to re-assign it to "itself" with another name